internal/observability/call.go

```diff
@@ -236,21 +236,19 @@ func (c call) endStats(
 ) {
 	c.edge.calls.Inc()
 
-	if c.direction == _directionInbound {
-		if deadlineTime, ok := c.ctx.Deadline(); ok {
-			c.edge.ttls.Observe(deadlineTime.Sub(c.started))
-		}
+	if deadlineTime, ok := c.ctx.Deadline(); ok {
+		c.edge.ttls.Observe(deadlineTime.Sub(c.started))
+	}
 
-		if res.requestSize > 0 {
-			c.edge.requestPayloadSizes.IncBucket(int64(res.requestSize))
-		}
+	if res.requestSize > 0 {
+		c.edge.requestPayloadSizes.IncBucket(int64(res.requestSize))
 	}
 
 	if res.err == nil && !res.isApplicationError {
 		c.edge.successes.Inc()
 		c.edge.latencies.Observe(elapsed)
 
-		if c.direction == _directionInbound && res.responseSize > 0 {
+		if res.responseSize > 0 {
```

Comment:
We should update the description of the metrics and mention that only request with no errors are being recorded.

Or can we record even errors?

Reply:
In case of errors, response typically would be very small compared to success response. Will update the documentation 👍